contrib/llvm-project-cmake/CMakeLists.txt

```diff
@@ -1,6 +1,5 @@
-if (APPLE OR SANITIZE STREQUAL "undefined" OR SANITIZE STREQUAL "memory")
-    # llvm-tblgen, that is used during LLVM build, doesn't work with UBSan.
-    set (ENABLE_EMBEDDED_COMPILER_DEFAULT OFF)
+if (APPLE OR SANITIZE STREQUAL "memory")
```

Comment:
not quite obvious why it was originally disabled
https://github.com/Algunenano/ClickHouse/commit/986bdf86ee7b8151e0f7d8d5f29f3ed41d61dff2